### PR TITLE
Stats token list

### PR DESCRIPTION
**Pull request by PawelKrawczyk96**

### CODE REVIEW

File: src/components/Stats/TokenListItem/TokenListItem.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import TokenListItem from '@components/Stats/TokenListItem/TokenListItem'
+import { BN } from '@project-serum/anchor'
+
+storiesOf('stats/TokenListItem', module)
+  .add('Item', () => {
+    let a = 9
```
Comment (luckmer) [inline src/components/Stats/TokenListItem/TokenListItem.stories.tsx]: declared a variable that is not used

File: src/components/PositionsList/Pagination/Pagination.tsx [hunk 2]
```diff
@@ -7,13 +7,19 @@ export interface IProps {
   pages: number
```
Comment (awojciak) [inline src/components/PositionsList/Pagination/Pagination.tsx:7]: maybe move Pagination out of position list folder

File: src/components/Stats/TokenListItem/TokenListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  return (
+    <Grid>
+      {displayType === 'tokens' ? (
+        <Grid
+          container
+          style={{ color: colors.white.main }}
+          classes={{ container: classes.container, root: classes.tokenList }}>
+          <Typography component='p'>{itemNumber}</Typography>
+          <Grid className={classes.tokenName}>
+            <img src={icons[icon]}></img>
```
Comment (awojciak) [inline src/components/Stats/TokenListItem/TokenListItem.tsx:45]: don't use icons object, we're using everywhere links from solana list

File: package.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@hookform/error-message": "^0.0.3",
     "@hookform/resolvers": "^0.1.0",
-    "@invariant-labs/sdk": "0.5.6",
+    "@invariant-labs/sdk": "0.5.7",
```
Comment (awojciak) [inline package.json:9]: revert this

**Review by awojciak — CHANGES_REQUESTED** [review]
also:
1) obraz
fix green values; also get rid of spaces, it would look better without them
2) 
obraz
i don't think that colors of this pagination are right

File: src/components/PositionsList/Pagination/Pagination.tsx (hunk 2 above)
Comment (PawelKrawczyk96) [reply src/components/PositionsList/Pagination/Pagination.tsx:7]: done

File: src/components/Stats/TokenListItem/TokenListItem.tsx (hunk 3 above)
Comment (PawelKrawczyk96) [reply src/components/Stats/TokenListItem/TokenListItem.tsx:45]: done

File: package.json (hunk 4 above)
Comment (PawelKrawczyk96) [reply package.json:9]: done

**Review by awojciak — APPROVED** [review]

**Pull request merged**